requirements.txt

```diff
@@ -24,31 +24,33 @@ MarkupSafe==1.0
 mistune==0.8.3
 nbconvert==5.3.1
 nbformat==4.4.0
-notebook==5.4.1
+notebook==5.7.8
 numpy==1.14.3
 pandas==0.22.0
 pandocfilters==1.4.2
 parso==0.2.0
 pexpect==4.5.0
 pickleshare==0.7.4
+prometheus-client==0.6.0
 prompt-toolkit==1.0.15
 ptyprocess==0.5.2
 pyasn1==0.4.2
 Pygments==2.2.0
 python-dateutil==2.7.2
 pytz==2018.4
-PyYAML==3.12
+PyYAML==3.13
```

Comment:
This version of PyYAML is still vulnerable. The advisory says to upgrade to _at least_ `4.2b1`

Reply:
Really? I wonder why it didn't upgrade all the way. Let me try again.

Reply:
I've removed pyyaml since this will form a base image. If a child image needs to open YAML files, it can be installed there and any security alerts would show up to be addressed there.